### PR TITLE
den-365 minor fix

### PR DESCRIPTION
**Pull request by harry-patel-jisc**

### CODE REVIEW

**Comment by harry-patel-jisc** [comment]
https://github.com/ajitjisc/moodle_dennis_plugin/blob/fc74483337e8855573e892b4da1c0ee1a79f6309/classes/task/run_sql_task.php#L40 should it be `>=` and not `=` ?

**Comment by harry-patel-jisc** [comment]
@ajitjisc please grant doug access to this repo and confirm you've seen the comment above

**Comment by ajitjisc** [comment]
> @ajitjisc please grant doug access to this repo and confirm you've seen the comment above

@harry-patel-jisc I have already sent an invite to Doug. Yes, I will change that condition soon.

**Comment by ajitjisc** [comment]
> query:
> 
> https://github.com/ajitjisc/moodle_dennis_plugin/blob/fc74483337e8855573e892b4da1c0ee1a79f6309/classes/task/run_sql_task.php#L40
> 
> should it be `>=` and not `=` ?

@harry-patel-jisc @doug-jisc , I have changed the condition here - > [https://github.com/ajitjisc/moodle_dennis_plugin/pull/6/commits/13f00b45a9f04f990adc008e3915b4d5b35832af](url) . Please take a look and review it.

File: classes/task/run_sql_task.php [hunk 1]
```diff
@@ -37,7 +37,7 @@ public function execute() {
             WHERE
                 log.origin = 'web' AND 
                 user.id <> 0 AND
-                DATE(FROM_UNIXTIME(log.timecreated)) = DATE_FORMAT(NOW() - INTERVAL 1 DAY, '%Y-%m-%d');"
+                DATE(FROM_UNIXTIME(log.timecreated)) >= DATE_FORMAT(NOW() - INTERVAL 1 DAY, '%Y-%m-%d');"
```
Comment (harry-patel-jisc) [inline classes/task/run_sql_task.php:40]: beginning to think your original condition was fine. Can you explain why you were happy to change it?

**Review by doug-jisc — APPROVED** [review]

**Comment by harry-patel-jisc** [comment]
@ajitjisc i can't approve this as i created it so feel to merge as Doug has approved

**Pull request merged**